### PR TITLE
Fix typo and cleanup

### PR DESCRIPTION
**Pull request by jboxman**
- https://github.com/openshift/openshift-docs/issues/22435

### CODE REVIEW

File: modules/images-create-metadata.adoc [hunk 1]
```diff
@@ -53,11 +49,7 @@ LABEL io.openshift.tags   mongodb,mongodb24,nosql
 ====
 
 |`*io.openshift.wants*`
-|Specifies a list of tags that the generation tools and the UI might use to
-provide relevant suggestions if you do not have the container images with given tags
-already. For example, if the container image wants `mysql` and `redis` and you
-do not have the container image with `redis` tag, then UI might suggest you to add
-this image into your deployment.
+|Specifies a list of tags that the generation tools and the UI might use to provide relevant suggestions if you do not have the container images with given tags already. For example, if the container image wants `mysql` and `redis` and you do not have the container image with `redis` tag, then UI might suggest you to add this image into your deployment.
```
Comment (kalexand-rh) [inline modules/images-create-metadata.adoc:52]: I think "given" is the wrong word. Maybe "required" or "specified?"

File: modules/images-create-metadata.adoc [hunk 2]
```diff
@@ -53,11 +49,7 @@ LABEL io.openshift.tags   mongodb,mongodb24,nosql
 ====
 
 |`*io.openshift.wants*`
```
Comment (kalexand-rh) [inline modules/images-create-metadata.adoc:51]: According to the current guidelines, these elements should be styled in just backticks: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

File: openshift_images/create-images.adoc [hunk 3]
```diff
@@ -1,18 +1,12 @@
...
-:toc-title:
-:prewrap!:
 toc::[]
 
 Learn how to create your own container images, based on pre-built images that are ready to help you.
-The process includes learning best practices for writing images,
-defining metadata for images, testing images and using a custom builder workflow to create images
- that can be used on
-{product-title}.
+The process includes learning best practices for writing images, defining metadata for images, testing images and using a custom builder workflow to create images that can be used on {product-title}.
```
Comment (kalexand-rh) [inline openshift_images/create-images.adoc:9]: s/testing images/testing images, 

maybe s/images that can be used on/images to use with

File: openshift_images/create-images.adoc [hunk 4]
```diff
@@ -1,18 +1,12 @@
...
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 Once you have created an image, you can push it to
```
Comment (kalexand-rh) [inline openshift_images/create-images.adoc:11]: s/Once you have created/After you create

File: openshift_images/create-images.adoc [hunk 5]
```diff
@@ -1,18 +1,12 @@
-:context: create-images
 [id="creating_images"]
```
Comment (kalexand-rh) [inline openshift_images/create-images.adoc:1]: The anchor should match the context, which usually matches the title: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata

File: modules/images-create-metadata.adoc (hunk 2 above)
Comment (jboxman) [reply modules/images-create-metadata.adoc:51]: @kalexand-rh, we seem to have quite a few of these in 22 files throughout the docs; I'll create a PR for that.

**Comment by jboxman** [comment]
@kalexand-rh, thanks!

**Pull request merged**